ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/views/TypedefEnumView.java

```diff
@@ -0,0 +1,4 @@
+package com.spectralogic.ds3autogen.c.views;
+
+public class TypedefEnumView {
```

Comment:
to keep the Helpers out of the Models, these views are intended to compose them and then be used by the templates... but I put that off until the bigger refactor was done and didn't mean to include this file yet.